src/styles/_section_image_and_text.scss

```diff
@@ -1,2 +1,8 @@
 // Image & Text
 // ======================================
+
+.text-image {}
```

Comment:
unused!

Reply:
There are a bunch of those, they'll get their styles soon ;)